### DISCUSSION TITLE
Approved icon tilting other icons

### DISCUSSION THREAD

**Discussion by RocketSheepOriginal** [ppy/osu#15991] [Q&A] [answered]
The icon in the top-left corner that indicates if the map is approved, graveyard or loved

If I change the beatmap to an approved one (correct icon ✔️) and quickly change to any other map, they will get tilted, since they lose orientation because of the appoved animation, every other icon (heart ❤️, ranked arrows ⏫ and question mark ❓) will get affected because their animations don't change the orientation.

If I change back to approved icon and leave the animation play completely, the icons orientation come back to normal.
This link is my video recording of this issue

https://user-images.githubusercontent.com/48867584/145127324-4823096a-3418-4a1b-b1cd-96abab328d87.mp4

**Comment by frenzibyte** [comment, marked as answer]
Wrong repo, this repo is for osu!(lazer) not osu!(stable). Report at https://github.com/ppy/osu-stable-issues/issues.

**Reply by peppy** [reply]
also please don't report, it's out of scope for fixing.
